### ISSUE TITLE
Test failure

### ISSUE THREAD

**Issue by hartmark** [lutris/lutris#3192]
Building the latest code yields a test failure.

```
(nosetests:42864): Gtk-WARNING **: 00:05:12.542: Theme parsing error: gtk.css:4:8: '' is not a valid color name
/home/markus/.cache/yay/lutris-git/src/lutris/lutris/gui/dialogs/__init__.py:10: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import GObject, Gtk, WebKit2
.......2020-10-11 00:05:13,202: MAME XML generation launched in the background, not returning anything this time
2020-10-11 00:05:13,203: Getting full game list from MAME...
2020-10-11 00:05:13,208: MAME isn't installed, can't retrieve systems list.
xrandr: Failed to get size of gamma for output screen
xrandr: Failed to get size of gamma for output screen
xrandr: Failed to get size of gamma for output screen
xrandr: Failed to get size of gamma for output screen
2020-10-11 00:05:13,774: Creating new configuration with runner linux
E/home/markus/.cache/yay/lutris-git/src/lutris/lutris/gui/application.py:69: Warning: g_set_application_name() called multiple times
  GLib.set_application_name(_("Lutris"))
2020-10-11 00:05:14,376: Creating new configuration with runner linux
............xrandr: Failed to get size of gamma for output screen
2020-10-11 00:05:15,858: The command "substitute" does not exist.
None
.xrandr: Failed to get size of gamma for output screen
.2020-10-11 00:05:15,866: This installer doesn't have a 'script' section
None
.xrandr: Failed to get size of gamma for output screen
.Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
.Terminated
.....2020-10-11 00:05:17,873: Migrating basetable field new_field
........................................................
======================================================================
ERROR: test_can_add_game (test_dialogs.TestGameDialog)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/markus/.cache/yay/lutris-git/src/lutris/tests/test_dialogs.py", line 105, in test_can_add_game
    game.remove(from_library=True)
TypeError: remove() got an unexpected keyword argument 'from_library'
-------------------- >> begin captured logging << --------------------
lutris.util.log: WARNING: MAME XML generation launched in the background, not returning anything this time
lutris.util.log: INFO: Getting full game list from MAME...
lutris.util.log: INFO: MAME isn't installed, can't retrieve systems list.
lutris.util.log: INFO: Creating new configuration with runner linux
--------------------- >> end captured logging << ---------------------

Name                                        Stmts   Miss  Cover
---------------------------------------------------------------
lutris/__init__.py                              1      0   100%
lutris/api.py                                 157    116    26%
lutris/cache.py                                20     12    40%
lutris/command.py                             157    116    26%
lutris/config.py                              146     25    83%
lutris/database/__init__.py                     0      0   100%
lutris/database/categories.py                  23     13    43%
lutris/database/games.py                      111     57    49%
lutris/database/schema.py                      41      0   100%
lutris/database/services.py                    22     13    41%
lutris/database/sources.py                     41     30    27%
lutris/database/sql.py                        117     18    85%
lutris/discord.py                              90     64    29%
lutris/exceptions.py                           24     10    58%
lutris/game.py                                446    318    29%
lutris/game_actions.py                        106     62    42%
lutris/gui/__init__.py                          0      0   100%
lutris/gui/application.py                     330    236    28%
lutris/gui/config/__init__.py                   2      0   100%
lutris/gui/config/add_game.py                  23      2    91%
lutris/gui/config/boxes.py                    454    168    63%
lutris/gui/config/common.py                   431    192    55%
lutris/gui/config/edit_game.py                 17     11    35%
lutris/gui/config/runner.py                    20     12    40%
lutris/gui/config/system.py                    19     11    42%
lutris/gui/dialogs/__init__.py                313    231    26%
lutris/gui/dialogs/cache.py                    36     27    25%
lutris/gui/dialogs/download.py                 32     23    28%
lutris/gui/dialogs/issue.py                    55     43    22%
lutris/gui/dialogs/log.py                      44     34    23%
lutris/gui/dialogs/runner_install.py          203    163    20%
lutris/gui/dialogs/runners.py                 156    119    24%
lutris/gui/dialogs/uninstall_game.py           73     58    21%
lutris/gui/installerwindow.py                 326    274    16%
lutris/gui/lutriswindow.py                    488    357    27%
lutris/gui/views/__init__.py                    2      0   100%
lutris/gui/views/base.py                       42     29    31%
lutris/gui/views/grid.py                       50     37    26%
lutris/gui/views/list.py                      123     97    21%
lutris/gui/views/media_loader.py               19     11    42%
lutris/gui/views/store.py                     104     50    52%
lutris/gui/views/store_item.py                 87     44    49%
lutris/gui/widgets/__init__.py                  0      0   100%
lutris/gui/widgets/cellrenderers.py            33     23    30%
lutris/gui/widgets/common.py                  200     75    62%
lutris/gui/widgets/contextual_menu.py          40     33    18%
lutris/gui/widgets/download_progress.py        78     68    13%
lutris/gui/widgets/game_bar.py                183    155    15%
lutris/gui/widgets/gi_composites.py            91     35    62%
lutris/gui/widgets/installer.py               317    257    19%
lutris/gui/widgets/log_text_view.py            53     43    19%
lutris/gui/widgets/searchable_combobox.py      56     40    29%
lutris/gui/widgets/sidebar.py                 192    146    24%
lutris/gui/widgets/status_icon.py              79     55    30%
lutris/gui/widgets/utils.py                   119     96    19%
lutris/gui/widgets/window.py                   30     24    20%
lutris/installer/__init__.py                   36     26    28%
lutris/installer/commands.py                  363    317    13%
lutris/installer/errors.py                     29      9    69%
lutris/installer/installer.py                 187    115    39%
lutris/installer/installer_file.py            114     89    22%
lutris/installer/interpreter.py               257    171    33%
lutris/installer/legacy.py                     14     12    14%
lutris/installer/steam_installer.py            78     57    27%
lutris/migrations/__init__.py                  17     10    41%
lutris/platforms.py                            10      0   100%
lutris/runner_interpreter.py                   78     69    12%
lutris/runners/__init__.py                     44     12    73%
lutris/runners/atari800.py                     75     50    33%
lutris/runners/browser.py                      18      5    72%
lutris/runners/commands/__init__.py             0      0   100%
lutris/runners/commands/dosbox.py              39     32    18%
lutris/runners/commands/wine.py               205    180    12%
lutris/runners/dolphin.py                      28     15    46%
lutris/runners/dosbox.py                       59     37    37%
lutris/runners/fsuae.py                        81     56    31%
lutris/runners/hatari.py                       67     48    28%
lutris/runners/json.py                         52     18    65%
lutris/runners/jzintv.py                       31     19    39%
lutris/runners/libretro.py                    172    126    27%
lutris/runners/linux.py                        64     41    36%
lutris/runners/mame.py                        126     76    40%
lutris/runners/mednafen.py                    149    128    14%
lutris/runners/mupen64plus.py                  28     13    54%
lutris/runners/o2em.py                         47     29    38%
lutris/runners/osmose.py                       20      9    55%
lutris/runners/pcsx2.py                        30      0   100%
lutris/runners/pico8.py                       143    100    30%
lutris/runners/redream.py                      42     27    36%
lutris/runners/reicast.py                      81     55    32%
lutris/runners/residualvm.py                   52     35    33%
lutris/runners/rpcs3.py                        21      8    62%
lutris/runners/runner.py                      295    179    39%
lutris/runners/scummvm.py                      69     41    41%
lutris/runners/snes9x.py                       37     18    51%
lutris/runners/steam.py                       192    100    48%
lutris/runners/vice.py                         93     63    32%
lutris/runners/web.py                          57     37    35%
lutris/runners/wine.py                        411    326    21%
lutris/runners/winesteam.py                   244    158    35%
lutris/runners/yuzu.py                         54     34    37%
lutris/runners/zdoom.py                        81     61    25%
lutris/runtime.py                             151    117    23%
lutris/services/__init__.py                     7      1    86%
lutris/services/base.py                       109     69    37%
lutris/services/gog.py                        272    197    28%
lutris/services/humblebundle.py               210    148    30%
lutris/services/lutris.py                      99     45    55%
lutris/services/service_game.py                63     33    48%
lutris/services/steam.py                      107     56    48%
lutris/services/xdg.py                         98     50    49%
lutris/settings.py                             30      0   100%
lutris/startup.py                              81     53    35%
lutris/sysoptions.py                           65      7    89%
lutris/util/__init__.py                        10      9    10%
lutris/util/audio.py                           11      7    36%
lutris/util/cookies.py                         39     35    10%
lutris/util/datapath.py                        19      5    74%
lutris/util/disks.py                           32     24    25%
lutris/util/display.py                        182    102    44%
lutris/util/downloader.py                     127    109    14%
lutris/util/extract.py                        179    155    13%
lutris/util/fileio.py                          24      2    92%
lutris/util/game_finder.py                     56     49    12%
lutris/util/gamecontrollerdb.py                43     28    35%
lutris/util/graphics/__init__.py                0      0   100%
lutris/util/graphics/displayconfig.py         281    121    57%
lutris/util/graphics/drivers.py                69     41    41%
lutris/util/graphics/glxinfo.py                40      6    85%
lutris/util/graphics/vkquery.py                42     24    43%
lutris/util/graphics/xephyr.py                  7      6    14%
lutris/util/graphics/xrandr.py                105     39    63%
lutris/util/http.py                           115     90    22%
lutris/util/jobs.py                            35     12    66%
lutris/util/joypad.py                          41     30    27%
lutris/util/libretro.py                        62     48    23%
lutris/util/linux.py                          307    125    59%
lutris/util/log.py                             22      1    95%
lutris/util/mame/__init__.py                    0      0   100%
lutris/util/mame/database.py                   52     38    27%
lutris/util/resources.py                        4      1    75%
lutris/util/settings.py                        25      9    64%
lutris/util/shell.py                           26     20    23%
lutris/util/steam/__init__.py                   0      0   100%
lutris/util/steam/appmanifest.py               83     53    36%
lutris/util/steam/config.py                    54     33    39%
lutris/util/steam/log.py                       38     33    13%
lutris/util/steam/vdf.py                       41     10    76%
lutris/util/strings.py                         88     28    68%
lutris/util/system.py                         239    136    43%
lutris/util/timer.py                           21     11    48%
lutris/util/wine/__init__.py                    0      0   100%
lutris/util/wine/cabinstall.py                154    125    19%
lutris/util/wine/dxvk.py                      149    105    30%
lutris/util/wine/fsync.py                      82     63    23%
lutris/util/wine/nine.py                       86     55    36%
lutris/util/wine/prefix.py                    146    117    20%
lutris/util/wine/registry.py                  249     61    76%
lutris/util/wine/wine.py                      272    147    46%
lutris/util/wine/x360ce.py                     71     54    24%
lutris/util/xdgshortcuts.py                    57     42    26%
lutris/util/yaml.py                            17      3    82%
---------------------------------------------------------------
TOTAL                                       15906  10132    36%
----------------------------------------------------------------------
Ran 86 tests in 7.339s

FAILED (errors=1)

```

**Comment by hartmark** [comment]
Seems the error was introduced in this commit

```
1 markus@bernard ..ode/aur-packages/lutris-git/src/lutris (git)-[bisect/bad~1|bisect] % git bisect bad
5c8469cca1b81b62b9c7fac0726988cb138656ad is the first bad commit
commit 5c8469cca1b81b62b9c7fac0726988cb138656ad
Author: Mathieu Comandon <strycore@gmail.com>
Date:   Fri Oct 9 22:04:42 2020 -0700

    Redesign uninstall game dialog (Closes #3100)

 lutris/game.py                           |  25 +----
 lutris/gui/dialogs/uninstall_game.py     | 163 ++++++++++++++++---------------
 lutris/runners/pico8.py                  |  11 ++-
 lutris/util/system.py                    |  25 +++--
 share/lutris/ui/dialog-uninstall-game.ui | 151 ----------------------------
 5 files changed, 111 insertions(+), 264 deletions(-)
 delete mode 100644 share/lutris/ui/dialog-uninstall-game.ui

```

**Comment by hartmark** [comment]
Pushed a fix in PR #3193

**Comment by hartmark** [comment]
commit from  #2945 fixes the test failure
